content/docs/2.0/scalers/metrics-api.md

```diff
@@ -0,0 +1,95 @@
++++
+title = "Metrics API"
+layout = "scaler"
+availability = "v2.0+"
+maintainer = "Community"
+description = "Scale applications based on a metric value from an API"
+go_file = "metrics_api_scaler"
++++
+
+This specification describes the `metrics-api` trigger that scales based on a metric value from an API.
+
+### Trigger prerequisites
+
+To use an external API as a source of metric you need to have a following endpoint:
+
+```yaml
```

Comment:
Let's add the whole OpenAPI spec here so that people can use it to validate their endpoint